test/parallel/test-fs-chmod-mask.js

```diff
@@ -89,6 +90,34 @@ function test(mode, asString) {
     fs.lchmodSync(link, input);
     assert.strictEqual(fs.lstatSync(link).mode & 0o777, mode);
   }
+
+  // Test that `lchmod` works on readonly (0x444) files.
+  if (fs.lchmod) {
```

Comment:
Please put this test case into another file - see the comment at the top of this file.

Reply:
Could be `/test/parallel/test-fs-chmod.js`? I'd prefer we don't create a new file.

Reply:
@refack Why not? That would reduce the dependencies among test cases and reduce the chance of flakes.